SETUP.md

```diff
@@ -0,0 +1,300 @@
+
+Git is used to track and manage changes of our code. Python is used in the pyOCD debugger as well as few other scripts and tools.
+
+<details><summary>Windows</summary>
+
+Make sure `python` and `git` are installed **and are available from Powershell**.
+
+You can use [this](https://www.python.org/downloads/) link to install Python 3. Select option 'Add Python to PATH' in the installer.
+
+To install Git, use [this](https://git-scm.com/downloads/win) link. Git will be added to PATH by default.
+
+</details>
+
+<details><summary>macOS</summary>
+
+Before proceeding, make sure `python3` and `git` commands are available in the terminal.
+
+Python and git can be installed with homebrew:
+```
+brew install git
+brew install python
+```
+
+</details>
+
+<details><summary>Linux</summary>
+
+Before proceeding, make sure `python3` and `git` commands are available in the terminal.
+
+On Ubuntu:
+```
+sudo apt update
+sudo apt install python3 python3-pip
+```
+
+On Fedora:
+```
+sudo dnf install python3 git
+```
+
+</details>
+
+### 2. Clone the repository
+
+You can clone the repo using SSH or HTTPS. If you want to push commits to Github, you must use SSH.
+
+
+<details><summary>SSH</summary>
+
+In the host terminal, open a folder with development projects. Then run:
+```sh
+git clone git@github.com:utat-ss/finch-flight-software.git
+cd finch-flight-software
+```
+</details>
+
+<details><summary>HTTPS</summary>
+
+In the host terminal, open a folder with development projects. Then run:
+```sh
+git clone https://github.com/utat-ss/finch-flight-software.git
+cd finch-flight-software
+```
+
+</details>
+
+
+### 3. Install pyOCD
+
+pyOCD is a tool that facilitates the interaction between the STM32 chip and your computer.
+
+In the host terminal, run:
+
+___________
+
+<details><summary>Windows</summary>
+
+`python -m pip install -U pyocd`
+
+</details>
+
+<details><summary>macOS/Linux</summary>
+
+`python3 -m pip install -U pyocd`
+
+</details>
+
+___________
+
+
+If you have a Raspberry Pi or non-x86 Windows platform to install pyOCD on, check out [this](https://pyocd.io/docs/installing_on_non_x86) pyOCD guide for more information.
+
+### 4. Download and place pyOCD packs
+
+pyOCD needs Keil packs for STM32G4 and STM32H7 to support our OBC and PAY boards. When you run any pyOCD command, you can pass a Keil pack with a `--pack` option. Our scripts also use `--pack` option and expect the packs under a specific path.
+
+Steps:
+
+1. Create folder named `packs` in the `finch-flight-software` folder.
+2. Open [this](https://www.keil.arm.com/packs/stm32h7xx_dfp-keil/overview/) page or search `stm32h7 pack keil`.
+3. Download the .pack file and place it in the `packs` folder.
+4. Rename the .pack file to `stm32h7.pack`.
+5. Open [this](https://www.keil.arm.com/packs/stm32g4xx_dfp-keil/overview/) page or search `stm32g4 pack keil`.
+6. Download the .pack file and place it in the `packs` folder.
+7. Rename the .pack file to `stm32g4.pack`.
```

Comment:
Already discussed in-person:
Having the required blobs under blobs/ (need to check their LICENSE)
might be better

Reply:
pack files are not binary files
west.yml might be the right place